examples/nextjs-with-typescript/README.md

```diff
@@ -24,8 +24,4 @@ or:
 
 [Next.js](https://github.com/zeit/next.js) is a framework for server-rendered React apps.
 
-## Troubleshooting
-
-### `Warning: Prop className did not match.`
-
-If you get this warning, please make sure that you configure `getInitialProps` in `pages/_document.tsx` correctly. Check the code in this example for more details.
+Additionally, the template project adds `@material-ui/core` and it's peer dependencies. As `@emotion` is used as a style engine in `@material-ui/core`, the necessary setup for it was done in the example project.
```

Comment:
Happy to add the change, however, should we do it to all the examples? Why only Next.js

Reply:
Let's add it everywhere. Will create dedicated PR to update all examples' README.md

Reply:
Opened https://github.com/mui-org/material-ui/pull/23938